pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -44,33 +45,35 @@ const (
 	monmapFile = "monmap"
 )
 
-func (c *Cluster) getLabels(daemonName string) map[string]string {
+func (c *Cluster) getLabels(daemonName string, canary bool, pvcName string) map[string]string {
 	// Mons have a service for each mon, so the additional pod data is relevant for its services
 	// Use pod labels to keep "mon: id" for legacy
 	labels := opspec.PodLabels(AppName, c.Namespace, "mon", daemonName)
 	// Add "mon_cluster: <namespace>" for legacy
 	labels[monClusterAttr] = c.Namespace
+	labels["mon_canary"] = strconv.FormatBool(canary)
```

Comment:
This isn't a big thing, but I think I did prefer the previous usage where the label `mon_canary=true` when the deployment was a canary and where the label was totally absent when it was not a canary. I think it makes it easier to see when something is a canary at a glance as a human. 

```suggestion
	if canary {
		labels["mon_canary"] = true
	}
```

Reply:
and what abut `pvc_name`? Should it be present only if it has a defined value?

Reply:
Yes the same pattern sounds good

Reply:
done! :-)

Reply:
LGTM! will merge after the CI completes